lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FixedStrategy.cxx

```diff
@@ -80,35 +76,35 @@ void FixedStrategy::updateBasis(const Point &,
   conservedPsi_k_ranks_.fill();
 }
 
-
 /* Virtual constructor */
 FixedStrategy * FixedStrategy::clone() const
 {
   return new FixedStrategy(*this);
 }
 
-
 /* String converter */
 String FixedStrategy::__repr__() const
 {
   return OSS() << "class=" << getClassName()
          << " derived from " << AdaptiveStrategyImplementation::__repr__();
 }
 
+/* involvesModelSelection accessor */
+Bool FixedStrategy::involvesModelSelection() const
```

Comment:
Yes I am. This is because `FixedStrategy` is not a model selection. When we use `LARS`, we feed it into a `LeastSquaresMetaModelSelectionFactory`, which _is_ a model selection:

https://github.com/openturns/openturns/blob/fe73b4d4102ee9cf35a3341b3b3c8d703a9f1081/lib/src/Base/Algo/LeastSquaresMetaModelSelectionFactory.cxx#L81

So the current code seems to be OK from my point of view.